website/server/controllers/api-v3/auth.js

```diff
@@ -239,17 +241,39 @@ function _passportFbProfile (accessToken) {
   });
 }
 
+function _passportGoogleProfile (accessToken) {
+  return new Bluebird((resolve, reject) => {
```

Comment:
using `Promise` instead of `Bluebird` results in an error.

    TypeError: A value undefined was yielded that could not be treated as a promise

What would the right way to do this be?